src/compiler/factory/emitHelpers.ts

```diff
@@ -783,7 +783,9 @@ namespace ts {
         text: `
             var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
                 if (k2 === undefined) k2 = k;
-                Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
+                var oldDesc = Object.getOwnPropertyDescriptor(m, k);
+                var desc = ("get" in oldDesc) && !("value" in oldDesc) ? oldDesc : { enumerable: true, get: function() { return m[k]; } };

```

Comment:
The correct code is actually:
```suggestion
                var oldDesc = m.__esModule && Object.getOwnPropertyDescriptor(m, k);
                var desc = oldDesc && "get" in oldDesc ? oldDesc
                    : { enumerable: true, get: function() { return m[k]; } };
```

Reply:
(Yeah, I had that for a while, but I wanted to play with the new feature that was added now, after verifying that `Object.freeze`-ing the `exports` make it possible to see that they're constants.)